save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/RegistrationView.kt

```diff
@@ -90,7 +92,8 @@ val registrationView: FC<RegistrationProps> = FC { props ->
             responseHandler = ::responseHandlerWithValidation,
         )
         if (response.ok) {
-            window.location.href = "${window.location.origin}/"
+            props.userInfoSetter(userInfo)
+            useNavigate(to = "/${FrontendRoutes.THANKS_FOR_REGISTRATION}")
             window.location.reload()
```

Comment:
do we need it taking into account that userInfoSetter is called?

Reply:
Just for beautiful loading animation so we will have a feeling that user really pressed a button